### ISSUE TITLE
Replying to older threads should render that reply in the timeline

### ISSUE THREAD

**Issue by bdimcheff** [bluesky-social/social-app#746]
**Describe the bug**

If you reply to one of your old posts it doesn't show up in anyone's home timelines.

**To Reproduce**

1. Go to your own profile, find a post from yesterday or something.
2. Reply to that old post, creating (or continuing) a thread.
3. Go to your home timeline. You won't see that reply. Nobody else will either.

**Expected behavior**

Show the reply to your thread, preferably with some context that it is a thread (potentially the last 1 or more posts). Twitter works this way.

twitter:
<img width="590" alt="Screenshot 2023-05-18 at 21 49 42" src="https://github.com/bluesky-social/social-app/assets/14111/73bfa135-43d4-41b4-9df8-7420ccc4a5c7">

this is what it could look like, where (part of) the thread is inserted where the _most recent post_ is, chronologically. In main, it's missing all the replies from my test account and just has my two posts from my main account.

<img width="588" alt="Screenshot 2023-05-18 at 21 53 20" src="https://github.com/bluesky-social/social-app/assets/14111/b8aeaf1c-faa5-49f3-906c-2d2d8de2f872">

**Screenshots**

**Details**

 - Platform: all apps built from this repo
 - Platform version:
 - App version:

**Additional context**

I'm going to open a PR in a sec that explains what I think is happening. It doesn't actually fix it fully, because I can't quite get there.

**Comment by teancom** [comment]
I have a long running thread listing out all of the books I've read this year, and my "work around" to this bug is to self-boost every reply as I make it, which *only* helps for those folks who don't have boosts disabled. Which, as someone who has boosts disabled, I completely understand why you would do that. But the alternative would be to quote skeet myself every time which... ick. Ick! So between this bug and the lack of being able to disable boosts per account followed vs. globally, I know there are people who are following me who have no idea about my book review thread.

**Comment by bdimcheff** [comment]
@teancom yeah this is close to what I'm trying to do (except with pictures of cats that i've pet)

**Comment by mary-ext** [comment]
Don't know how I missed this one issue when I filed the PR but this PR adds bumping to replies
https://github.com/bluesky-social/social-app/pull/2604
